pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -273,9 +273,16 @@ type VCPUs struct {
 	VCPU []VCPUsVCPU `xml:"vcpu"`
 }
 
+type CPUModel struct {
+	Fallback string `xml:"fallback,attr,omitempty"`
+	VendorID string `xml:"vendor_id,attr,omitempty"`
+	Value    string `xml:",chardata"`
+}
+
 type CPU struct {
+	Check    string       `xml:"check,attr,omitempty"`
```

Comment:
Is this value populated somewhere in the converter?

Reply:
Currently there is none, but libvirt domain XML has this value.